tests/unit/hca/test_accession_mapper.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+from assertpy import assert_that
+
+from hca.accession_mapper import AccessionMapper
+from hca.submission import HcaSubmission
+from tests.unit.utils import make_ingest_entity, random_id, random_uuid
+
+
+class MyTestCase(unittest.TestCase):
```

Comment:
could rename to AccessionMapperTest

Reply:
Yep. Tx. Sorry about that.

Reply:
I merged it, but going to fix this in the next PR.